### ISSUE TITLE
Prepublish checklist is advising that the background image is too small on the page

### ISSUE THREAD

**Issue by bmattb** [GoogleForCreators/web-stories-wp#5374]
## Bug Description

Added image to a page, set the image as background but prepublish checklist shows that the Image is too small on the page. (seems to be based on the size on page PRIOR to setting as background)

## Steps to Reproduce

1. Place an image on the page
2. Drag the image to the edge of the workspace to designate it as a background image
3. Prepublish check will indicate the error message noted.

## Screenshots
![Screen Shot 2020-11-20 at 3 39 34 PM](https://user-images.githubusercontent.com/58738530/99847880-c489c380-2b46-11eb-877c-8f374015e8bd.png)

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by zachhale** [comment]
So this investigation highlighted to me a quirk (which others may already be aware of) with background images in that when they are made into a background image they retain their existing bounding rect/positioning info which is then used to re-place the image when it's detached.

Because of this, both `mediaElementSizeOnPage` (this bug) and the related `mediaElementResolution` have to work differently for background images. For size on page, I solved this by just ignoring background images for this error since they're handled differently. For `mediaElementResolution`, though, we'd need to calculate resolution differently than we are currently doing for background images since the bounding rect info is incorrect.

I see two paths for this:
1. Ignore background images in `mediaElementSizeOnPage` and create new ticket to calculate `mediaElementResolution` differently for background images
2. When images are attached as background images, we re-calculate the bounding rect, then as part of detaching, we do some sort of automatic placement like if we were to pull the image from the media library.

I think (1) would be the ideal path at least right now and is how I implemented this PR.

Thoughts? @swissspidy @carlos-kelly @o-fernandez
